src/db.py

```diff
@@ -6,6 +6,7 @@
 from pymongo import MongoClient
 from pbkdf2 import PBKDF2
 import subprocess
+import jwt
```

Comment:
a HA! I found the culprit for why the tests broke!

When you add a new external library, make sure to put it in requirements.txt. I got this one, but watch out for that one in the future as well!